core/lib/refinery/core/configuration.rb

```diff
@@ -68,12 +58,8 @@ def clear_stylesheets!
         self.stylesheets = []
       end
 
-      def dragonfly_custom_backend?
-        config.dragonfly_custom_backend_class.present?
-      end
-
       def dragonfly_custom_backend_class
-        config.dragonfly_custom_backend_class.constantize if dragonfly_custom_backend?
+        raise "Refinery::Dragonfly now handles all dragonfly configuration. Consult 'config/initializers/refinery/dragonfly.rb'."
```

Comment:
I keep my comment about "You should deprecate this method instead of raising an error." :)

Reply:
Sorry - didn't read it properly last time. Done now.